AppTests/Services/SQLiteRepositoryTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using App.Services;
+
+namespace AppTests.Services
+{
+    public class SQLiteRepositoryTests : IDisposable
+    {
+        private readonly SQLiteRepository _repository;
```

Comment:
You chose to use underscores for naming here. Is this an XUnit naming convention?

Unless there is a good reason for this, I think we should keep names consistent and remove the underscores (`_`).